autonomy/configurations/base.py

```diff
@@ -366,12 +366,6 @@ def try_to_process_nested_fields(  # pylint: disable=too-many-locals
                         nested_override_key,
                         nested_override_value,
                     ) in nested_value.items():
-                        if (
-                            nested_override_key
-                            not in config_class.NESTED_FIELDS_ALLOWED_TO_UPDATE  # type: ignore
-                        ):
-                            raise ValueError("Trying to override non-nested field.")
```

Comment:
We don't need to check for `NESTED_FIELDS_ALLOWED_TO_UPDATE`, we're planning to remove this variable in one of the coming releases